internal/executor/executor.go

```diff
@@ -174,10 +174,20 @@ func (executor *Executor) RunBuild(ctx context.Context) {
 	}
 
 	if hasWaitForTerminalInstruction {
+		expireIn := 15 * time.Minute
+
+		expireInString, ok := executor.env["CIRRUS_TERMINAL_EXPIRE_IN"]
```

Comment:
Maybe call it `CIRRUS_TERMINAL_INACTIVITY_WAIT_PERIOD`? 🤔

Reply:
Renamed in https://github.com/cirruslabs/cirrus-ci-agent/pull/192/commits/2019305d0104e0fea7df126587b9b3c7de25b55b.